src/openrct2/actions/BannerSetNameAction.cpp

```diff
@@ -55,6 +56,12 @@ GameActions::Result::Ptr BannerSetNameAction::Query() const
 GameActions::Result::Ptr BannerSetNameAction::Execute() const
 {
     auto banner = GetBanner(_bannerIndex);
+    if (banner == nullptr)
+    {
+        log_warning("Invalid banner id, banner id = %d", _bannerIndex);
+        return MakeResult(GameActions::Status::InvalidParameters, STR_NONE);
```

Comment:
I know this is just copied, but we should have a proper error message and not a blank one.

Reply:
Again will be addressed in follow on PRs. This PR is purely just pulling the commits from the NSF.